src/main/index.js

```diff
@@ -31,7 +31,7 @@ function runApp() {
       },
       {
         label: 'Open in a New Window',
-        visible: parameters.linkURL.includes((new URL(browserWindow.webContents.getURL())).origin),
+        visible: parameters.linkURL.startsWith(browserWindow.webContents.getURL()),
```

Comment:
Probably should add code comment to document this pitfall

Reply:
Added a comment now.